docs/get_started/plan_deployment/traditional_deployment/cluster_consideration/clus_plan_esm.md

```diff
@@ -1,36 +1,36 @@
+-   When you set up security in a cluster to use an external security manager, ensure that you review and, if required, configure security on each node in the cluster, as described in Security Access Manager.
+-   If you change the external security manager configuration after the initial setup, first make the changes in the `wkplc\_comp.propreties` file on the primary node of the cluster. If additional nodes exist in the cluster, ensure that your changes to the `wkplc\_comp.properties` file on the primary node are propagated to the `wkplc\_comp.properties` files on other nodes in the cluster.
 
 ## Security Access Manager cluster considerations
 
 -   Ensure that you run the validate-pdadmin-connection task on each node in the cluster.
--   If the validate-pdadmin-connection task fails, run the run-svrssl-config task before attempting to run validate-pdadmin-connection again. Note that the wp.acc.impl.PDServerName parameter in the wkplc\_comp.properties file represents an individually configured AMJRTE connection to Security Access Manager, and each node in the cluster must have a unique value for wp.acc.impl.PDServerName before running the run-svrssl-config task.
--   If you are using an external Web server, additional configuration is required before running any task to configure an external security manager with a HCL Portal cluster. Edit the wkplc\_comp.properties file on each node, and ensure that the values for the wp.ac.impl.JunctionHost and wp.ac.impl.JunctionPort properties are set to the backend server host name and port number you are using for your Web server.
--   Ensure that the WebSEAL Trust Association Interceptor \(TAI\) parameters, found in the wkplc\_comp.properties file, are the same on each node in the cluster. If you run a configuration task that overwrites the WebSEAL junction, the WebSphere® Application Server TAI properties are not automatically updated. Therefore, you must manually ensure that all nodes are using the same parameters. To manually ensure the nodes are the same, use the Deployment Manager WebSphere Integrated Solutions Console and go to **Security** \> **Global security** \> **Web and SIP Security** \> **Trust Association** \> **Interceptors** \> **com.ibm.sec.authn.tai.TAMETai** \> **Custom properties**.
+-   If the validate-pdadmin-connection task fails, run the run-svrssl-config task before you attempt to run the validate-pdadmin-connection task again. !!!note that the `wp.acc.impl.PDServerName` parameter in the `wkplc\_comp.properties` file represents an individually configured AMJRTE connection to Security Access Manager, and each node in the cluster must have a unique value for the  `wp.acc.impl.PDServerName` parameter before you run the run-svrssl-config task.
```

Comment:
```suggestion
-   If the validate-pdadmin-connection task fails, run the run-svrssl-config task before you attempt to run the validate-pdadmin-connection task again. Note that the `wp.acc.impl.PDServerName` parameter in the `wkplc\_comp.properties` file represents an individually configured AMJRTE connection to Security Access Manager, and each node in the cluster must have a unique value for the `wp.acc.impl.PDServerName` parameter before you run the run-svrssl-config task.
```
!!!note will not work if within a paragraph

Reply:
Thanks. Good to know.